repos/system_upgrade/common/actors/removeleftoverpackages/actor.py

```diff
@@ -0,0 +1,22 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import removeleftoverpackages
+from leapp.models import LeftoverPackages, RemovedPackages
+from leapp.tags import IPUWorkflowTag, RPMUpgradePhaseTag
+
+
+class RemoveLeftoverPackages(Actor):
+    """
+    Remove packages left on the system after the upgrade to higher major version of RHEL.
+
+    Removal of packages is necessary in order to keep the machine in supported state.
+    Produce a message that is consumed by the `reportleftoverpackages` actor,
+    which reports on the packages that have been removed.
+    """
+
+    name = 'remove_leftover_packages'
+    consumes = (LeftoverPackages, )
+    produces = (RemovedPackages, )
+    tags = (RPMUpgradePhaseTag, IPUWorkflowTag, )
```

Comment:
@tomasfratrik ah. sorry. I actually missed that you still removed the experimental tag. this actor must not be executed without user explicitely allowing it to execute.

Reply:
Added it Back